### ISSUE TITLE
As a user I would like to run packageless in a container

### ISSUE THREAD

**Issue by everettraven** [everettraven/packageless#52]
packageless needs an official Docker image. This image should be built and published to DockerHub as part of the packageless release process using a GitHub Actions workflow.

**Comment by DipayanP007** [comment]
Hey @everettraven can you assign me this? I can work on this

**Comment by everettraven** [comment]
> Hey @everettraven can you assign me this? I can work on this

@DipayanP007 sure thing!

**Comment by DipayanP007** [comment]
hey @everettraven can you provide me with the name of the image to be built?

**Comment by DipayanP007** [comment]
and as I can see there is no Dockerfile. it would be good if you can provide me with the commands to build the project so that I can write the dockerfile too.

**Comment by everettraven** [comment]
> hey @everettraven can you provide me with the name of the image to be built?

@DipayanP007 The image name should be **packageless**

One thing to note is that this app requires access to the Docker daemon/socket. I don't think this should have any effect on the Dockerfile though.

**Comment by DipayanP007** [comment]
yeah it won't be a problem for Dockerfile. We will only need to explicitly mount the docker socket while running the container. So can you give the commands to run the project?

**Comment by everettraven** [comment]
> yeah it won't be a problem for Dockerfile. We will only need to explicitly mount the docker socket while running the container. So can you give the commands to run the project?
> 
> 

@DipayanP007 To build the project you can run `go build`.

That should produce a binary file named packageless (packageless.exe if on Windows).

You can run the program with ./packageless

I was thinking it would be best to have that as an ENTRYPOINT though as there are many sub commands that can be run.

Here is a link to the documentation that you can browse for a bit more information (specifically the CLI section for info on sub commands ): https://everettraven.github.io/packageless/

**Comment by everettraven** [comment]
Hey @DipayanP007 I just wanted to check in and see if this is still in progress?

**Comment by everettraven** [comment]
@DipayanP007 Are you still interested in working on this?

If not I will un-assign you on this issue so that someone else can pick it up.

Thanks!

**Comment by DipayanP007** [comment]
Hey @everettraven Sorry I have been a bit busy. You can unassign this

**Comment by everettraven** [comment]
@DipayanP007 No worries, thanks!
